tests/golos.vesting_tests.cpp

```diff
@@ -85,7 +85,7 @@ class golos_vesting_tester : public golos_tester {
         const string unknown_asset    = amsg("unknown asset");
         const string wrong_precision  = amsg("wrong asset precision");
         const string vesting_params   = amsg("not found vesting params");
-        const string issuer_not_autority = "missing authority of issuer";
+        const string issuer_not_autority = "missing authority of " + cfg::emission_name.to_string();
```

Comment:
I've planned to create parametrized errors for such cases, so we can check like `err.issuer_not_auth(name)`. But it's ok for now.